redirects/7.1/concepts/auth-kerberos-clientconfig.html

```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/auth-sync/
```

Comment:
auth-sync isn't a folder it is a markdown page

Reply:
This is correct - the `.md` files in the code are actually published to what look like folder paths in the live site .. so this page is redirected to https://docs.alfresco.com/content-services/7.1/admin/auth-sync/. HTH